src-electron/rest/initialize.js

```diff
@@ -147,23 +153,27 @@ function sessionCreate(db) {
       } else {
         pkgArray = []
       }
-
+      let partitionsLength =
```

Comment:
All top-level json files are a partition. so the zcl and template json files are each a partition. Those partitions are used between zcl and template relationship based on category so that generation does not overflow across zigbee and matter